lib/matplotlib/tests/test_backend_ps.py

```diff
@@ -148,3 +149,11 @@ def test_determinism_all():
 def test_determinism_all_tex():
     """Test for reproducible PS/tex output"""
     _determinism_check(format="ps", usetex=True)
+
+
+@image_comparison(baseline_images=["empty"], extensions=["eps"])
+def test_transparency():
+    fig, ax = plt.subplots()
+    ax.set_axis_off()
+    ax.plot([3, 2, 1], alpha=0)
+    ax.text(.5, .5, "foo", alpha=0)
```

Comment:
I would advise to put the text in the drawn area ([0, 2]x[1, 3], +/- the default margins). Something like `ax.text(0.5, 1.5, "foo", alpha=0)`. Otherwise, the text instance is not visible, even with a non null alpha value, is it not?

Reply:
good catch